### ISSUE TITLE
Install

### ISSUE THREAD

**Issue by bessermt** [Microsoft/cpprestsdk#184]
I'm new and looking to check out this project. 

https://github.com/Microsoft/cpprestsdk/wiki/Getting-Started-Tutorial

I'm using NuGet and I'm search for "Casablanca" as instructed and getting "No packages found". 

I'm using Visual Studio 2015, if that matters. 

How can I install?

**Comment by Danimoth** [comment]
Hi! We have just updated the Getting Started Tutorial. The new screenshots are from Visual Studio 2015. Can you try following the updated tutorial?

In particular for being unable to find Casablanca: 
- At the top left, is "Browse" the selected tab?
- At the top right, Is the package source "nuget.org"?

**Comment by bessermt** [comment]
There was no way to select the package source. It showed nuget.org**** ("****" meaning additional stuff afterwards I don't recall right now). I researched the issue and it looks like a bug in VS2015 that NuGet searches don't work. The solution is to uninstall and reinstall NuGet. Then it seems to work. The install issue I have now is that I have a long list of cpprestsdk's to chose from and the descriptions are identical. I'm planning to try it out on Windows 7 and so I'm guessing I want "cpprestsdk.v140.windesktop.msvcstl.dyn.rt-dyn", but the descriptions are all identical. I suspect the "...winapp..." versions are for running on a Windows Surface device (something I know almost nothing about). Might be worth mentioning something on each download so it's clearer which one to select. 

Anyway, thanks and take all that for what it's worth. :-)

**Comment by bessermt** [comment]
Here's some links you might want to consider:

A document about the bug:
http://docs.nuget.org/release-notes/known-issues#error-installing-packages-with-nuget-3.4,-3.4.1

I know not Casablanca's problem, but it made me assume Casablanca was discontinued as a product and I was about 10 minutes away from giving up on it. 

-----

Might consider updating the following. It also makes it sound like Casablanca is no longer available, especially since the link leads to the old project location:

https://msdn.microsoft.com/en-us/library/jj969455.aspx

Cheers.

**Comment by Danimoth** [comment]
Hi!
Thanks for reporting your findings! 

> NuGet bug

I added the link to the "known NuGet issues" in the Getting Started page. 

> MSDN change

Good catch. Indeed, this is outdated and I will look into changing it. We switched to github "recently" and there are still some links that point to our old repository. Furthermore, cpprestsdk is still supported. I will make sure MSDN reflects that and I am glad that you didn't give up on us!

>Which NuGet Package to use?

For quick reference of the packages mentioned below, here is the web version of the cpprestsdk packages:
https://www.nuget.org/packages?q=cpprestsdk
(These are the same results that you get in Visual Studio)

The package you want for desktop (Windows Vista and above) is indeed:
**cpprestsdk.v140.windesktop.msvcstl.dyn.rt-dyn**

There is a separate package for xp in the list. The package:
**cpprestsdk.v140.winapp.msvcstl.dyn.rt-dyn**
is for Windows Store apps. Maybe having "Store" in the name would have been better. We will consider it for future releases.
